src/pages/LoginPage/components/Login/Login.jsx

```diff
@@ -1,72 +1,124 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState } from "react";
 
-import './Login.css';
-import wavingPikachu from '../../../../img/waving-pikachu.png';
-import thorEndgame from '../../../../img/thor-endgame.png';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faUser } from '@fortawesome/free-solid-svg-icons';
-import { faLock } from '@fortawesome/free-solid-svg-icons';
-import { faFacebookF } from '@fortawesome/free-brands-svg-icons';
-import { faGoogle } from '@fortawesome/free-brands-svg-icons';
-import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
-import { auth, gProvider, fProvider } from '../../../../fire';
-import { useDataLayerValue } from '../../../../context/DataLayer';
+import "./Login.css";
+import wavingPikachu from "../../../../img/waving-pikachu.png";
+import thorEndgame from "../../../../img/thor-endgame.png";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faUser } from "@fortawesome/free-solid-svg-icons";
+import { faLock } from "@fortawesome/free-solid-svg-icons";
+import { faFacebookF } from "@fortawesome/free-brands-svg-icons";
+import { faGoogle } from "@fortawesome/free-brands-svg-icons";
+import { faEnvelope } from "@fortawesome/free-solid-svg-icons";
+import { auth, gProvider, fProvider } from "../../../../fire";
+import { useDataLayerValue } from "../../../../context/DataLayer";
+import API from "../../../../utils/API";
 
 function Login() {
   const [{}, dispatch] = useDataLayerValue();
   const [elementsObj, setElementsObj] = useState({});
   const [userObj, setUserObj] = useState({});
 
   const addSignUpMode = () => {
-    elementsObj.container.classList.add('sign-up-mode');
+    elementsObj.container.classList.add("sign-up-mode");
   };
 
   const remSignUpMode = () => {
-    elementsObj.container.classList.remove('sign-up-mode');
+    elementsObj.container.classList.remove("sign-up-mode");
   };
 
-  const handleSignUp = e => {
+  const handleSignUp = (e) => {
     e.preventDefault();
+
     auth
       .createUserWithEmailAndPassword(userObj.email, userObj.password)
-      .then(user => {
+      .then((user) => {
         // signed up
         var u = auth.currentUser;
         u.updateProfile({
-          displayName: userObj.displayName
+          displayName: userObj.displayName,
         })
           .then(() => {
+            API.signUpUser({ u })
```

Comment:
what does u mean here? you should use full names so other people know what's going on

Reply:
done